modules/host/negotiate.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/NebulousLabs/Sia/modules"
 )
 
+// acceptNegotiation writes the 'accept' response to conn.
+func acceptNegotiation(conn net.Conn) error {
+	return encoding.WriteObject(conn, modules.AcceptResponse)
+}
```

Comment:
do we want to move this into modules/negotiate.go? Then the renter would have access to it as well.